### PR TITLE
Add seed database for combined hg38/hg19 db

### PR DESCRIPTION
**Pull request by SRodenburg**
Add seed files and some documentation for hg38.

### CODE REVIEW

**Comment by khzhu** [comment]
hi @SRodenburg , tested. hg19 reference genes are missing from the reference_genome_gene table. Please fix that. Thank you!
```
mysql> select distinct rg.name from reference_genome_gene rgg join reference_genome rg on rg.reference_genome_id = rgg.reference_genome_id;
+------+
| name |
+------+
| hg38 |
+------+
1 row in set (0.00 sec)
```

**Comment by khzhu** [comment]
we should have genes from hg19 and hg38 reference genomes, so that hg19 datasets can co-exists with those hg38 datasets in the same portal installation.

**Comment by SRodenburg** [comment]
Hi @khzhu 
Although I see the use of having hg19 genes next to hg38 genes in the database, this was was really not in the scope of this PR. Here I rather want to add the data to allow deployment of a hg38 instance. 
Since Entrez IDs and Hugo symbols are conflicting between hg19 and 38, simply adding hg38 genes to the hg19 database proved not so straight-forward. Did you ever manage to build a hybrid database?

**Comment by khzhu** [comment]
Hi @SRodenburg , at PMC we have hg19 and hg38 datasets stored in the same portal database. Entrez IDs and Hugo symbols are not conflicting between hg19 and 38, although hg38 might have some Hugo symbols that are not included in the hg19 build or visa versa. 
The difference between the two are mainly cytogenetic information such as cytoband, gene start/end positions. If you take a look at the database schema, you can see reference_genome_gene table is linked with the gene table.

**Comment by khzhu** [comment]
I personally think it is more useful to have a database that supports both genome builds.

File: seedDB_hg38/cgds.sql [hunk 1]
```diff
@@ -0,0 +1,849 @@
+--
```
Comment (inodb) [inline seedDB_hg38/cgds.sql]: is there a difference in this cgds sql compared to hg37?

**Comment by inodb** [comment]
@SRodenburg could you maybe elaborate what the specific issues were with trying to import both in the same database. We could file that as an issue for someone else to work on, maybe @khzhu can give some pointers, since it sounds like she's already solved that issue

File: seedDB_hg38/cgds.sql (hunk 1 above)
Comment (SRodenburg) [reply seedDB_hg38/cgds.sql]: There is none, but I guess there is between versions. Thought it would be good to put the tested version with the seed data.

**Comment by SRodenburg** [comment]
@khzhu @inodb Kelsey was right.  hg19/hg38 are now both included.

**Comment by inodb** [comment]
Thanks @SRodenburg - looks like the tests are failing cause of the classic CircleCI issue where it tries to run the tests under the thehyve org

**Comment by inodb** [comment]
@khzhu you think this is ok to merge?

**Comment by inodb** [comment]
@SRodenburg So sorry, one more thing actually: do you think we should get rid of hg19 only database completely?

**Comment by khzhu** [comment]
@SRodenburg , please let me know when new hg19/hg38 seed database is available at data hub for testing. thanks!

**Comment by SRodenburg** [comment]
@khzhu I think it is in my last commit

**Comment by khzhu** [comment]
@SRodenburg , the download link points to datahub though, 
```
https://github.com/cBioPortal/datahub/blob/c5d7103674e17b1b8a144973364de4f349ac8988/seedDB_hg38/seed-cbioportal_hg38_v2.11.0.sql.gz
```
can you please update that. Thanks!

**Comment by khzhu** [comment]
ping @SRodenburg , can you please update the link to download the seed database? thanks!

**Comment by SRodenburg** [comment]
Hi @khzhu I am currently out of the office until coming Tuesday. Will do it ASAP.

**Comment by SRodenburg** [comment]
@khzhu I think this is what you want. Can you confirm? I renamed the hg19/hg38 seed DB and moved it to the appropriate directory.

**Comment by khzhu** [comment]
hi @SRodenburg , @inodb 
I am sorry for the delayed response. I tested and confirmed that the seed database supports both hg19 and hg38. Thank you!
```
mysql> select distinct name from reference_genome_gene rgg join reference_genome rg on rgg.reference_genome_id = rg.reference_genome_id;
+------+
| name |
+------+
| hg19 |
| hg38 |
+------+
```

**Comment by SRodenburg** [comment]
@khzhu Great!
@inodb Can be merged?

**Comment by inodb** [comment]
Hmmm looks like this has the classic CircleCI issue where the tests are enabled under the thehyve org causing a failure, because it's not running under cBioPortal org. Gonna merge and 🤞

**Comment by inodb** [comment]
@SRodenburg I did notice this is all still under hg38_seedDB folder. We can prolly make that a single folder now that cgds & seedDB are exactly the same? I'll merge this, but we can follow up with that in a separate PR: https://github.com/cBioPortal/datahub/issues/991

**Pull request merged**